src/databricks/labs/ucx/source_code/linters/pyspark.py

```diff
@@ -353,3 +319,59 @@ def _find_matcher(self, node: NodeNG):
         if matcher is None:
             return None
         return matcher if matcher.matches(node) else None
+
+
+class SparkSqlPyLinter(PythonLinter, Fixer):
+
+    _PREFIX = "spark-sql-"
+
+    def __init__(self, sql_linter: SqlLinter, sql_fixers: list[Fixer]):
+        self._sql_linter = sql_linter
+        self._sql_fixers = sql_fixers
+
+    def can_fix(self, advice_code: str) -> bool:
+        return advice_code.startswith(self._PREFIX)
+
+    def lint_tree(self, tree: Tree) -> Iterable[Advice]:
+        for call_node, query in self._visit_call_nodes(tree):
+            for value in InferredValue.infer_from_node(query):
+                if not value.is_inferred():
+                    yield Advisory.from_node(
+                        code=f"{self._PREFIX}cannot-autofix-table-reference",
```

Comment:
```suggestion
                        code=f"cannot-autofix-table-reference",
```
it's still the same problem code.